### PR TITLE
feat: separate CLI Tutor Mode helptext

### PR DESCRIPTION
**Pull request by jessicaschilling**
Closes https://github.com/ipfs-shipyard/ipfs-webui/issues/1686.

This PR separates out the i18n key `cliModal.extraNotes` so we can display different CLI Tutor Mode modal helptext in different situations:
- "IPFS Config" modal (already existed)
- "Set Pinning" command in Files screen context menu (new in this PR)

**To do:**
The i18n keys are split out, but I'm missing something that's probably obvious in terms of getting helptext to display when a tutor-mode modal is invoked from the context menu. @lidel or @rafaelramalho19, do you mind a quick look if you get the chance? 🙏

### CODE REVIEW

**Review by rafaelramalho19 — APPROVED** [review]

**Comment by jessicaschilling** [comment]
Thanks, @rafaelramalho19, but can you please look at the "to do" item when you get a chance? Thank you!

**Comment by rafaelramalho19** [comment]
@jessicaschilling by help text you mean the `title` html attribute?

**Comment by jessicaschilling** [comment]
@rafaelramalho19 - cliModal.ExtraNotes is customized as part of this work (that's the helptext), but I can't get it to display conditionally for the "Set pinning" command under the "three dots" context menu.

**Comment by jessicaschilling** [comment]
Thanks, @rafaelramalho19!

@lidel - can you please have a look when you get the chance? Would be one more item done in the pinning epic ...

File: public/locales/en/files.json [hunk 1]
```diff
@@ -19,6 +19,9 @@
     "copyHash": "Copy CID",
     "share": "Share link"
   },
+  "cliModal": {
+    "extraNotes": "This command is only for local pinning/unpinning. If you want to use a remote pinning service, consult that service's documentation to learn how to pin/unpin from the command line."
```
Comment (lidel) [inline public/locales/en/files.json:23]:
```suggestion
    "extraNotesPinning": "This command is only for local pinning/unpinning. If you want to use a remote pinning service, consult that service's documentation to learn how to pin/unpin from the command line."
```

File: public/locales/en/settings.json [hunk 2]
```diff
@@ -19,6 +19,9 @@
   },
   "apiDescription": "<0>If your node is configured with a <1>custom API address</1>, including a port other than the default 5001, enter it here.</0>",
   "cliDescription": "<0>Enable this option to display a \"view code\" <1></1> icon next to common IPFS commands. Clicking it opens a modal with that command's CLI code, so you can paste it into the IPFS command-line interface in your terminal.</0>",
+  "cliModal": {
+    "extraNotes": "If you've made changes to the config in this page's code editor that you'd like to save, click the download icon next to the copy button to download it as a JSON file."
```
Comment (lidel) [inline public/locales/en/settings.json:23]:
```suggestion
    "extraNotesJsonConfig": "If you've made changes to the config in this page's code editor that you'd like to save, click the download icon next to the copy button to download it as a JSON file."
```

**Review by lidel — CHANGES_REQUESTED** [review]
LGTM but we should make key names more explicit, 
will save us time when debugging bugs in the future :)

@jessicaschilling mind updating places where below keys are used?

**Comment by jessicaschilling** [comment]
@lidel - good call. Updated. Please merge at your leisure ...

**Pull request merged**